addons/mail/static/src/core/common/chat_hub.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
 <t t-name="mail.ChatHub">
-    <div class="o-mail-ChatHub" t-if="isShown or ui.isSmall">
+    <div class="o-mail-ChatHub" t-if="!store.discuss?.isActive or ui.isSmall">
```

Comment:
The goal of using `isShown` and overriding it in `public_web` was that `discuss` makes no sense in other contexts (in particular in portal/livechat), so we should keep this behavior.

Reply:
`isShown` getter is misleading because this is ignored in mobile with the condition here. This was actually meant for disabling chat bubbles, which is also not desired with current code changes.

If we keep the getter, this is just for the sake to return true and override in `public_web` which are currently the only user of `store.discuss`. This is quite a lot of indirection for just that, when this could be immediately explicit in template where needed. And this is less changes to add more discuss app users.

I understand reducing the bundle size in different contexts, but there's a maintenance cost. And the bundle size is a factor when this is a big contribution, e.g. lots of discuss code that is unused in website.

Reply:
Is it just isShown = true now?
If so, it should be removed from line 11 of template.

But it's still not clean to reference discuss when the field/model are not even defined in this bundle.